### ISSUE TITLE
Add Docker file

### ISSUE THREAD

**Issue by impredicative** [jantic/DeOldify#9]
Can a Docker file and a Docker image with a pretrained model be provided for use with nvidia-docker? The idea is to make it possible to colorize easily.

**Comment by jantic** [comment]
I'll keep this open with the intention of getting that done.  I agree- it should be done!  It'll be slightly lower on my priority list (no guarantees on timeline) but it'll get done.

**Comment by AlexanderKozhevin** [comment]
@jantic @impredicative 
I launched it inside of a docker container. And even created image from it.  The base images was nvidia/cuda.
But there are 2 issues.
1. I was able to launch it only on version 7.5 of cuda. All other version provided the same error 
```
CUDA driver version is insufficient for CUDA runtime version
```
I thought it's related to host OS cuda version, but looks like not. 
2. I was able to commit container and created an image from it. But after launching new container from it I got this error once again.
```
CUDA driver version is insufficient for CUDA runtime version
```
Which is super strange, taking into account that it's exactly the same code with the same versions.
